config/webpacker.yml

```diff
@@ -11,7 +11,7 @@ default: &default
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  resolved_paths: []
+  resolved_paths: ["node_modules/govuk-frontend/govuk"]
```

Comment:
@dankmitchell - does adding the resolved path render the CopyWebpackPlugin useless (hence why it was removed?)

Reply:
Correct